openapi.yml

```diff
@@ -4941,6 +4977,54 @@ components:
                         colour: RED
                         last_name: some text
                         first_name: some text
+        NotificationsCreateParams:
+            description: ''
+            required:
+                - target
+                - message
+                - channels
+            type: object
+            properties:
+                target:
+                    description: Creates a notification for signins
+                    type: string
+                message:
+                    description: ''
+                    type: string
+                signin_filters:
+                    $ref: '#/components/schemas/NotifcationsSigninFilterParams'
+                    description: ''
+                channels:
```

Comment:
This is to pass 'EMAIL' and/or 'SMS', correct?

Reply:
@fernandojinzenji That's correct

Reply:
If there is a range of limited values allowed as the `channel`, it should probably be defined as an enum